component-samples/demo/README.MD

```diff
@@ -299,7 +299,7 @@ Finally, the mTLS enabled cURL request will be:
     --cacert scripts/secrets/ca-cert.pem --cert scripts/api-user.pem \
      --location --request POST 'http://host.docker.internal:8039/api/v1/rvinfo' \
     --header 'Content-Type: text/plain' \
-    --data-raw '[[[5,"{rv-dns}"],[3,8040],[12,1],[2,"{rv-ip}"],[4,8040]]]'
+    --data-raw '[[[5,"\{rv-dns}"],[3,8040],[12,1],[2,"\{rv-ip}"],[4,8040]]]'
```

Comment:
<img width="538" alt="image" src="https://github.com/fido-device-onboard/pri-fidoiot/assets/52932399/250d4109-4fe3-43b7-b7ea-a34b268b50bf">
Why do we need this change ?

Reply:
This change is made according to this JIRA issue - https://jira.devtools.intel.com/browse/SDOPT-12786

Reply:
Backslash is not required and adds more confusion.

Reply:
@Sai-Anudeep47 Alright should I close this PR ?

Reply:
Yes, we can close this